source/postprocess/velocity_statistics.cc

```diff
@@ -66,21 +57,14 @@ namespace aspect
         {
           if (cell->is_locally_owned())
             {
-              fe_values_for_rms.reinit (cell);
-              fe_values_for_rms[this->introspection().extractors.velocities].get_function_values (this->get_solution(),
-                  velocity_values_for_rms);
-              fe_values_for_max.reinit (cell);
-              fe_values_for_max[this->introspection().extractors.velocities].get_function_values (this->get_solution(),
-                  velocity_values_for_max);
-              for (unsigned int q = 0; q < n_q_points_for_rms; ++q)
-                {
-                  local_velocity_square_integral += ((velocity_values_for_rms[q] * velocity_values_for_rms[q]) *
-                                                     fe_values_for_rms.JxW(q));
-                }
-
-              for (unsigned int q = 0; q < n_q_points_for_max; ++q)
+              fe_values.reinit (cell);
+              fe_values[this->introspection().extractors.velocities].get_function_values (this->get_solution(),
+                                                                                          velocity_values);
+              for (unsigned int q = 0; q < n_q_points; ++q)
                 {
-                  local_max_velocity = std::max (std::sqrt(velocity_values_for_max[q]*velocity_values_for_max[q]),
+                  local_velocity_square_integral += ((velocity_values[q] * velocity_values[q]) *
+                                                     fe_values.JxW(q));
+                  local_max_velocity = std::max (std::sqrt(velocity_values[q]*velocity_values[q]),
```

Comment:
Same outcome, but perhaps slightly easier to read:
```suggestion
                  local_max_velocity = std::max ((velocity_values[q].norm(),
```